com.unity.shadergraph/Editor/Importers/ShaderSubGraphImporter.cs

```diff
@@ -73,6 +73,16 @@ public override void OnImportAsset(AssetImportContext ctx)
             Texture2D texture = Resources.Load<Texture2D>("Icons/sg_subgraph_icon@64");
             ctx.AddObjectToAsset("MainAsset", graphAsset, texture);
             ctx.SetMainObject(graphAsset);
+
+            var metadata = ScriptableObject.CreateInstance<ShaderGraphMetadata>();
```

Comment:
I think I meant this to be ShaderSubGraphMetadata, need to fix

Reply:
fixed